README.md

```diff
@@ -7,37 +7,39 @@ These benchmarks are based on a similar project by
 
 _**Note**: at the moment the ExCALIBUR benchmarks are a work-in-progress._
 
+## Before you start
+
+If you are using spack for other projects, you might have to clear the cache if it exists:
+`rm -rf ~/.spack/cache`
+
 ## Requirements
 
 ### Spack
 
-_**Note**: we require Spack 0.18.  In some HPC facilities there may be already a
-central Spack installation available.  In principle you should be able to use a
-system installation of Spack (you only need to have `spack` in the `PATH`), but
-you need to make sure the version is recent enough to install the required
-packages and understand the provided environments.  Instructions below show you
-how to install Spack locally._
+_**Note**: In some HPC facilities there may be already a central Spack installation available.
```

Comment:
Uhm, we do require at least Spack 0.18: #59.  Why was that removed?  All what we do should also be compatible with 0.19 and 0.20, that may be made clearer.

Reply:
It's safer to install the latest version anyway, so they don't have to worry about which one is installed on their system.